backend/model/model.go

```diff
@@ -2779,48 +2842,79 @@ func (obj *Alert) sendSlackAlert(ctx context.Context, db *gorm.DB, alertID int,
 				"click",
 				slack.NewTextBlockObject(
 					slack.PlainTextType,
-					caser.String(strings.ToLower(titleStr))+" Error",
+					caser.String(strings.ToLower(titleStr)),
 					false,
 					false,
 				),
 			)
 			button.URL = routing.AttachQueryParam(ctx, errorLink, "action", strings.ToLower(string(action)))
-			actionBlock = append(actionBlock, button)
+			actionBlocks = append(actionBlocks, button)
 		}
 
 		snoozeButton := slack.NewButtonBlockElement(
 			"",
 			"click",
 			slack.NewTextBlockObject(
 				slack.PlainTextType,
-				"Snooze Error",
+				"Snooze",
 				false,
 				false,
 			),
 		)
 		snoozeButton.URL = routing.AttachQueryParam(ctx, errorLink, "action", "snooze")
-		actionBlock = append(actionBlock, snoozeButton)
+		actionBlocks = append(actionBlocks, snoozeButton)
 
-		blockSet = append(blockSet, slack.NewActionBlock(
-			"",
-			actionBlock...,
-		))
-		blockSet = append(blockSet, slack.NewDividerBlock())
-		msg.Attachments = []slack.Attachment{
-			{
-				Color:  "#961e13",
-				Blocks: slack.Blocks{BlockSet: blockSet},
-			},
+		var stackTrace []*modelInputs.ErrorTrace
+		var stackTraceBlock *slack.TextBlockObject
+
+		stackTraceString := input.ErrorObject.MappedStackTrace
+		if stackTraceString == nil {
+			stackTraceString = input.ErrorObject.StackTrace
+		}
+
+		if err := json.Unmarshal([]byte(*stackTraceString), &stackTrace); err == nil {
+			firstTrace := stackTrace[0]
+
+			var fileLocation string
+			if firstTrace.LineNumber != nil {
+				fileLocation = fmt.Sprintf("%s:%d", *firstTrace.FileName, *firstTrace.LineNumber)
+			} else if firstTrace.FileName != nil {
+				fileLocation = *firstTrace.FileName
+			} else {
+				fileLocation = "File unknown"
+			}
+
+			if len(fileLocation) > FILE_NAME_LENGTH_LIMIT {
+				cutoffIndex := len(fileLocation) - FILE_NAME_LENGTH_LIMIT
+				substringIndex := strings.Index(fileLocation[cutoffIndex:], "/")
+				if substringIndex > -1 {
+					finalCutoffIndex := cutoffIndex + substringIndex
+					fileLocation = "..." + fileLocation[finalCutoffIndex:]
+				}
+			}
+
+			stackTraceBlock = slack.NewTextBlockObject(slack.MarkdownType, fileLocation, false, false)
+		}
+
+		bodyBlockSet = append(bodyBlockSet, slack.NewSectionBlock(eventBlock, nil, nil))
+		bodyBlockSet = append(bodyBlockSet, slack.NewSectionBlock(errorMessageBlock, nil, nil))
+		bodyBlockSet = append(bodyBlockSet, slack.NewSectionBlock(sessionBlock, nil, nil))
+		bodyBlockSet = append(bodyBlockSet, slack.NewActionBlock("", actionBlocks...))
+		if stackTraceBlock != nil {
+			highlightLogo := *slack.NewImageBlockElement("https://app.highlight.io/logo192.png", "Highlight logo")
+			bodyBlockSet = append(bodyBlockSet, slack.NewContextBlock("", highlightLogo, stackTraceBlock))
 		}
 	case AlertType.NEW_USER:
```

Comment:
I don't believe this is working - can create a card to investigate and fix separately

Reply:
yes pls! new user alerts altogether?

Reply:
I have to dig a little deeper, but saw the last new user alert we received in #new-user-alerts was in early September

https://github.com/highlight/highlight/issues/6835